robocorp-code/src/robocorp_code/inspector/web/_web_inspector.py

```diff
@@ -141,6 +141,7 @@ def page(self, auto_create) -> Optional[Page]:
         self.loop()
 
         page = self._page
+        log.debug("Page is: {page}")
```

Comment:
This should probably not be here (I did remove this message, but it's back in this PR).